cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -774,6 +774,72 @@ class TranslatedIfStmt extends TranslatedStmt, ConditionContext {
   }
 }
 
+class TranslatedConstExprIfStmt extends TranslatedStmt, ConditionContext {
+  override ConstexprIfStmt stmt;
```

Comment:
This is now pretty much a copy/paste of the `IfStmt` class with the exception of this one line. We could probably introduce an abstract class that contained both `ConstexprIfStmt` and `IfStmt` to get rid of this duplication, but it's probably not worth it for this one case

Reply:
Yeah, indeed. Would be nice to do the same thing for `IfStmt` and `ConstexprIfStmt` too actually.